### DISCUSSION TITLE
Tooltips/hover cards are stuck visible

### DISCUSSION THREAD

**Discussion by zbjornson** [community/community#66403] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

As of today, tooltips aren't reliably hiding after mousing-off the hover target throughout GitHub. Here's an example from the community discussions page, but it also affects PRs, issues, etc.
![stuck-tooltips](https://github.com/community/community/assets/469365/6d10cb9b-7d48-4c1a-ad01-137755fa65ae)

**Comment by thestumbler** [comment]
Me too. This suddenly started overnight, and it makes the site almost unusable.

**Comment by corang** [comment]
Same here, firefox. Same issue happened earlier this year too 🤦‍♂️

**Reply by corang** [reply]
Seems to be the same issue as https://github.com/orgs/community/discussions/44630

**Comment by Michael-Kruggel** [comment]
Happening to me too on Chrome.

**Comment by oleg-derevenetz** [comment]
Upvoting. Brave browser (Chromium-based).

**Comment by Rhys-T** [comment]
I noticed it earlier today, but it seems to have gone back to working properly just now (after a reload), at least on my machine. Can anyone else confirm?

**Reply by thestumbler** [reply]
Indeed. I am no longer annoyed.
